src/Pages/Playground/components/playgrounApiDocs.tsx

```diff
@@ -0,0 +1,587 @@
+                      max={localProperties[property]?.maximum}
+                      min={localProperties[property]?.minimum}
+                      enumArray={
+                        localProperties[property]?.enum ||
+                        localProperties[property]?.items?.enum
+                      }
+                      defaultValue={localProperties[property]?.default}
+                      examples={localProperties[property]?.examples}
+                      description={localProperties[property]?.description}
+                      items={localProperties[property]?.items}
+                      minItems={localProperties[property]?.minItems}
+                      maxItems={localProperties[property]?.maxItems}
+                      pattern={localProperties[property]?.pattern}
+                      additionalProperties={
+                        localProperties[property]?.additionalProperties
+                      }
+                      buttonClick={() =>
+                        setShowSignatureBox(!showSignatureBox())
+                      }
+                    />
+                  </>
+                )}
+              </For>
+            </Show>
+            <Show when={!localProperties && content.enum}>
+              <Signature
+                type={content?.type}
+                enumArray={content?.enum}
+                items={content?.items}
+                pattern={content?.pattern}
+                defaultValue={content?.default}
+                additionalProperties={content?.additionalProperties}
+                buttonClick={() => setShowSignatureBox(!showSignatureBox())}
+              />
+            </Show>
+          </div>
+        </Show>
+        <Show when={showSourceButton()}>
+          <JsonSchema json={content} mouseHoverFunc={mouseHoverFunc} />
+        </Show>
+      </div>
+      <div class="end"></div>
+    </>
+  );
+};
+const Signature: Component<SignatureType> = ({
+  property,
+  title,
+  type,
+  max,
+  min,
+  defaultValue,
+  examples,
+  description,
+  enumArray,
+  items,
+  minItems,
+  maxItems,
+  pattern,
+  additionalProperties,
+  buttonClick,
+}) => {
+  const [showSignature, setShowSignature] = createSignal(false);
+  const [activeClass , setActiveClass] = createSignal('');
+  const strDescription = description?.replace(/(`.`)/g, match => `<code class="bg-[rgba(255,255,255,.16)] text-[#fff] text-[14px] mr-1.5 px-2 py-1.5 font-normal rounded">${match}</code>`)?.replace(/`/g ,'');
+  const signatureExpand = () => {
+    buttonClick();
+    setShowSignature(!showSignature());
+    if( activeClass() === ''){
+      setActiveClass('box-active')
+    } else {
+      setActiveClass('')
+    }
+  };
+
+  const signatureType =
+    items?.additionalProperties !== undefined && property.items?.title
+      ? property?.items?.title
+      : additionalProperties !== undefined && property.title
+      ? property.title
+      : Array.isArray(type)
+      ? type?.join(", ")
+      : type;
```

Comment:
I see a lot of chained ternaries here. I wonder if you could convert to if else or switch cases for the sake of readablity?

Reply:
Yeah, you're right. I will do this.